defaults/main.yml

```diff
@@ -4,7 +4,10 @@ ssmtp:
   # User and group that will be able to use ssmtp
   group: ~
   user: ~
-
+  restrict: true
```

Comment:
Can you alphabetically order these vars? So it should be:

```YAML
ssmtp:
  auth: true
  # User and group that will be able to use ssmtp
  group: ~
  restrict: true
  revaliases: true
  use_inventory_hostname: false
  user: ~
```

Reply:
Should be okey now